game/client/gui_helpers/stack_menu.lua

```diff
@@ -23,12 +25,52 @@ function StackMenu:create(yStart, overlay, gap, title)
     return vals
 end
 
-function StackMenu:nextWidgetPosition()
-    local position = getWidgetCentre(WIDGET_WIDTH, self.y)
-    self.y = self.y - self.widgetGap
+function StackMenu:nextWidgetPosition(width, centered, x, y)
+    local position
+    if(centered) then 
```

Comment:
In Lua it is more common to have the if statements without the brackets (As they are not needed, similar to python etc)

If it is ok, these should be removed throughout the file 😃